spec/features/idv/analytics_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+feature 'Analytics Regression', js: true do
+  include IdvStepHelper
+
+  let(:user) { user_with_2fa }
+  let(:fake_analytics) { FakeAnalytics.new }
+  # rubocop:disable Layout/LineLength
+  let(:happy_path_events) do
+    {
+      'IdV: intro visited' => {},
+      'IdV: doc auth welcome visited' => { flow_path: 'standard', step: 'welcome', step_count: 1 },
+      'IdV: doc auth welcome submitted' => { success: true, errors: {}, flow_path: 'standard', step: 'welcome', step_count: 1 },
+      'IdV: doc auth agreement visited' => { flow_path: 'standard', step: 'agreement', step_count: 1 },
+      'IdV: doc auth agreement submitted' => { success: true, errors: {}, flow_path: 'standard', step: 'agreement', step_count: 1 },
+      'IdV: doc auth upload visited' => { flow_path: 'standard', step: 'upload', step_count: 1 },
+      'IdV: doc auth upload submitted' => { success: true, errors: {}, destination: :document_capture, flow_path: 'standard', step: 'upload', step_count: 1 },
```

Comment:
love seeing all these listed out!! I think they'll be a great way for us to try finding a way to move the doc auth events to the typed, documented ones in `analytics_events.rb`!!! lmk if you'd like to chat about ways we could try to define them more explicitly! (like maybe each flow step has an `analytics_method` property or something, would love to discuss ideas for how we could incorporate that with the JS frontend logging controller as well